src/pages/TeamOverviewPage.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import TeamLeadCard from 'components/TeamLeadCard';
+import TeamMembersList from 'components/TeamMembersList';
+import useFetchTeamOverview from 'hooks/useFetchTeamOverview';
+import {useLocation, useParams} from 'react-router-dom';
+import {Spinner} from 'components/Spinner';
+import SearchInput from 'components/SearchInput';
+import Header from '../components/Header';
+import {Container} from '../components/GlobalComponents';
+
+const TeamOverviewPage = (): JSX.Element => {
```

Comment:
what do you think about becoming the Page components name more explicit? Like on this one